components/pickerbutton/CHANGELOG.md

```diff
@@ -208,128 +254,110 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 <a name="4.0.1"></a>
+
 ## 4.0.1
+
 🗓
 2023-08-03 • 📝 [Commits](https://github.com/adobe/spectrum-css/compare/@spectrum-css/pickerbutton@4.0.0...@spectrum-css/pickerbutton@4.0.1)
 
 **Note:** Version bump only for package @spectrum-css/pickerbutton
 
 <a name="4.0.0"></a>
+
 ## 4.0.0
+
 🗓
 2023-07-31 • 📝 [Commits](https://github.com/adobe/spectrum-css/compare/@spectrum-css/pickerbutton@3.0.34...@spectrum-css/pickerbutton@4.0.0)
 
 \*feat(pickerbutton)!: migrate to use spectrum tokens (#1940)([ad6051d](https://github.com/adobe/spectrum-css/commit/ad6051d)), closes[#1940](https://github.com/adobe/spectrum-css/issues/1940)
 
-    	###
-    	🛑 BREAKING CHANGES
+### 🛑 BREAKING CHANGES
 
-    		*
-    		migrates the Picker Button component to use `@adobe/spectrum-tokens`
+#### Migrates the Picker Button component to use `@adobe/spectrum-tokens`
 
 - feat(pickerbutton)!: update to use tokens
-
-- feat(pickerbutton): updating tokens
-
 - chore(pickerbutton): move css from generated to index
-
-- feat(pickerbutton): updating with tokens
-
 - docs(pickerbutton): update docs html to correct icon sizes
-
-- refactor(pickerbutton): fixing variable names and tokens
-
-- chore(pickerbutton): update story
-
 - chore(pickerbutton): update storybook controls and template
-
 - chore(pickerbutton): adding more storybook stories for express, quiet etc
-
 - fix(pickerbutton): fixing icon color
-
-- chore(pickerbutton): reset yarn file
-
-- chore: reset yarn file
-
 - refactor(pickerbutton): fix button fill padding calculations
-
 - chore(pickerbutton): remove invalid and isKeyboardFocused variants
-
 - chore(pickerbutton): prevent focused and open when disabled
-
 - fix(pickerbutton): fix disabled hover
-
-update mods
-
 - fix(searchwithin): pass through mod for picker button border color
-
-- chore(pickerbutton): bumping up tokens release
-
 - chore(pickerbutton): update token peer dependency
-
-- fix(searchwithin): update searchwithin story to match docs site
-
-use picker instead of pickerbutton
-remove extra border
-
-- chore(pickerbutton): manual version increase for beta release
-
-- chore(pickerbutton): remove pickerbutton-generated css
-
-- fix(pickerbutton): remove icononly class and remove padding from uiicononly
-
-removes padding from uiicononly class to allow for larger icons in slots
-
+- fix(searchwithin): update searchwithin story to match docs site  
+  use picker instead of pickerbutton and remove extra border
+- fix(pickerbutton): remove icononly class and remove padding from uiicononly  
+  removes padding from uiicononly class to allow for larger icons in slots
 - fix(pickerbutton): explicitly add box-sizing border-box
-
 - fix(pickerbutton): fix icon size
 
-- chore(pickerbutton): manual version increase for beta release
+### Migration guide
+
+#### .spectrum-PickerButton-UIIcon class removed
+
+The `.spectrum-PickerButton-UIIcon` class no longer matches our naming convention. Both types of icons now use the `spectrum-PickerButton-icon` class
```

Comment:
Is this bit of the migration guide here specifically because these changes were made in version 3.0.34? How did you know where to put this section, as well as the other section about Quiet variant?

Reply:
I looked at the git blame for the addition of the docs and found which release they were added in (also this is under version 4.0).